action.yml

```diff
@@ -15,6 +15,7 @@ inputs:
   auto_fix:
     description: "Auto-fix code formatting errors"
     required: false
+    default: "false"
```

Comment:
Shouldn't we turn it on by default in pipelines too?  That will result in the better error "we could have fixed this" right?

Reply:
Autofix is enabled by default. I meant to remove this line after my latest refactor. Defaulting this variable as either true or false changes the natural flow of the goat. Default is scan once for initial and twice if errors that can be autofixed. Set auto_fix=true to bypass the initial scan and only autofix. Set auto_fix=false to only run an initial scan with no autofix.